packages/governance/src/committee.js

```diff
@@ -17,6 +17,12 @@ import { QuorumRule } from './question.js';
 
 const { ceilDivide } = natSafeMath;
 
+/**
+ * @typedef {{
+ * castBallotFor: (questionHandle: Handle<'Question'>, positions: Position[]) => Promise<void>
```

Comment:
@Chris-Hibbert I just realized that in order for committee members to be able to put questions as well, the "charter" contract should be a different shape:

Our design for raising debt limits involved using a method on a public facet, which assumed users had access to `zoe` and `E(zoe).getPublicFacet()`.
https://github.com/Agoric/inter-protocol-bootstrap/blob/56de1ed1821e19828a6f74ff10b7fce27b3f2cb7/vote-raise-debt-limit.js#L177

I think life will be simpler if the new charter contract you're building uses methods on an offer result instead of methods on a public facet.

Reply:
> ... if the new charter contract you're building ...

There isn't a task corresponding to that description on my radar. Can we make an issue?
I presume we want a variant of econCommitteeCharter that serves PSM only. If the capability to invoke votes is carried by an offer result, should the various committee members each invoke a separate invitation, or is an invitation exercised once in bootstrap, and the offer result handed out from there?

Reply:
> Can we make an issue?

Yes, we can; I more or less expected that was part of "Do The Right Thing" that you said what you had what you need for. The capability to vote is taken care of by the committee contract. My concern here is for posing questions. You're right that this raises a question of how the invitations get to the committee members. I suppose we can use the same mechanism as we did for the voting invitations, but it's worth some thought / discussion.